### PR TITLE
update busco to version 5.0

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
@abretaud its just an update and does not include the new `Metaeuk` stuff. But also not sure if needed. We would need this tool for the VGP project. Could you please have a look?

File: tools/busco/macros.xml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0"?>
 <macros>
-    <token name="@TOOL_VERSION@">4.1.4</token>
+    <token name="@TOOL_VERSION@">5.0.0</token>
+    <token name="@VERSION_SUFFIX@">0</token>
 
     <xml name="citations">
         <citations>
             <citation type="doi">10.1093/bioinformatics/btv351</citation>
+            10.1007/978-1-4939-9173-0_14
```
Comment (nsoranzo) [inline tools/busco/macros.xml:9]:
```suggestion
            <citation type="doi">10.1007/978-1-4939-9173-0_14</citation>
```

File: tools/busco/busco.xml [hunk 2]
```diff
@@ -23,112 +24,114 @@ busco
 --lineage_dataset '${lineage_dataset}'
 --update-data
 --mode '${mode}'
--o busco_galaxy
+--out busco_galaxy
 --cpu \${GALAXY_SLOTS:-4}
 --evalue ${adv.evalue}
 ${adv.long}
 --limit ${adv.limit}
+--augustus
```
Comment (nsoranzo) [inline tools/busco/busco.xml:32]: Should this be a user-selectable parameter?

File: tools/busco/macros.xml (hunk 1 above)
Comment (bgruening) [reply tools/busco/macros.xml:9]: Oh, I was thinking to include that citation but I think the former is enough and more accessible.

File: tools/busco/busco.xml (hunk 2 above)
Comment (bgruening) [reply tools/busco/busco.xml:32]: I don't know there is a second option offered (Metaeuk) which we haven't included yet but I think it's only used for metagenonics? It's not clear to me.

File: tools/busco/busco.xml [hunk 3]
```diff
@@ -7,128 +7,260 @@
...
-        </param>
+        <param type="data" name="input" format="fasta" label="Sequences to analyse" help="Can be an assembled genome or transcriptome (DNA), or protein sequences from an annotated gene set." />
+        <conditional name="busco_mode">
+            <param argument="--mode" type="select" label="Mode">
+                <option value="geno">Genome assemblies (DNA)</option>
+                <option value="tran">Transcriptome assemblies (DNA)</option>
+                <option value="prot">annotated gene sets (protein)</option>
+            </param>
+            <when value="geno">
+                <conditional name="use_augustus">
```
Comment (bgruening) [inline tools/busco/busco.xml:67]: to my understanding, the Augustus/Metaeuk option is only relevant in the genome case ...

File: tools/busco/busco.xml [hunk 4]
```diff
@@ -7,128 +7,260 @@
...
-                    <param name="augustus_model" type="data" format="augustus" label="Augustus model"/>
-                </when>
-                <when value="builtin">
-                    <param name="augustus_species" type="select" label="Augustus species model">
-                        <expand macro="augustus_species"/>
-                    </param>
-                </when>
-            </conditional>
-            <param argument="--long" type="boolean" checked="false" truevalue="--long" falsevalue="" label="Optimization mode Augustus self-training" help="Adds considerably to run time, but can improve results for some non-model organisms"/>
+            <param argument="--evalue" type="float" value="0.001" min="0" max="1" label="E-value cutoff for BLAST searches." />
```
Comment (bgruening) [inline tools/busco/busco.xml:103]: I changed the default value here - to be inline with the documentation

**Review by mvdbeek — APPROVED** [review]
Thanks @bgruening!

**Comment by bgruening** [comment]
Thanks for the review!

**Comment by abretaud** [comment]
Cool stuff, thanks for the update! (I was afk)

**Pull request merged**